ktxspec.adoc

```diff
@@ -651,6 +651,12 @@ uses global data. The format of the global data for a scheme
 is specified in the reference given in the _Global Data Format_
 column of <<supercompressionSchemes>>.
 
+Image data may be conditioned with Rate-Distortion Optimization
+prior to supercompression to improve compressibility, further
+reducing the final size. This is an encoding detail which is
+transparent to the user of the texture so KTX provides no indication
+that RDO has been used.
```

Comment:
nit: Although there's no explicit indication, `KTXwriterScParams` metadata may provide some hints.

Reply:
I've reworked the RDO description in b364d9f.